tutorial/consuming_packages/different_configurations.rst

```diff
@@ -45,6 +45,19 @@ folder:
 
     # The default profile can also be checked with the command "conan profile show"
 
+.. note::
+
+    **Using Powershell**
+
+    For Powershell users it is recommended to modify the profile so it is no longer needed to use the
+    ``--conf=tools.env.virtualenv:powershell=True`` command. Open your profile file an add:
+
+    .. code-block:: bash
+
+        [conf]
+        tools.env.virtualenv:powershell=True
+
+    This only modifies the chosen profile and further profiles created will require the same change.
```

Comment:
Maybe we can move this to the _use tools as conan packages_ section so that users find it earlier. I would modify the other note, adding this information, a bit summarized and adding a link to the [profile section](https://docs.conan.io/2/reference/config_files/profiles.html#reference-config-files-profiles)